kaniko-example/README.md

```diff
@@ -30,7 +30,11 @@ This is an example showing how to collect build-info, while using the Kaniko con
 
     ```console
```

Comment:
Let's add the line that defines the ```IMAGE_TAG``` variable here with ```latest``` as the value. If we don't, I'm not sure this example will work.